.packit.yml

```diff
@@ -0,0 +1,40 @@
+upstream_package_name: subscription-manager
+downstream_package_name: subscription-manager
+specfile_path: subscription-manager.spec
+
+srpm_build_deps:
+  - python3-devel
+  - openssl-devel
+  - gcc
+  - python3-setuptools
+  - gettext
+  - glib2-devel
+  - system-release
+  - python3-dateutil
+  - 'pkgconfig(systemd)'
+  - tito
+
+actions:
+  get-current-version:
+    - awk '/^Version:/ {print $2;}' ./subscription-manager.spec
+  create-archive:
+    - bash -c 'tito build --tgz -o ./'
+    - bash -c 'ls -1 ./subscription-manager-*.tar.gz'
+  fix-spec-file:
+    - echo 'nothing to fix'
```

Comment:
I tried with another tito-based source, and it seems packit does everything properly on its own:
- it creates the source tarball
- it modifies the version and the release in the spec file

So my suggestion here is to simply drop all the actions, and there should be a git-based version.

Reply:
I also tried to use packit without `actions`, but it does not work, when I try to build RPMs locally (`packit build locally`) or in mock (`packit build in-mock`).

BTW: Current solution is not perfect in this PR, because it does not create RPMs with unique NVR. I'm working on the fix.

Reply:
Ah, the pattern of the tags includes also the name, and thus by default it is handled as part of the version string.

Simply tell packit what's the pattern for the tags, as I did in the packit Fedora configuration [1]
```yaml
upstream_tag_template: "subscription-manager-{version}-1"
```
With that config key and no action, everything seems to be working fine for me:
- the version does not include "subscription-manager"
- the resulting srpm of `packit srpm` seems correct
- `packit build locally` works too

[1] https://src.fedoraproject.org/rpms/subscription-manager/blob/rawhide/f/packit.yaml